### DISCUSSION TITLE
Preparing build machine failed 🥲

### DISCUSSION THREAD

**Discussion by romanfurman6** [codemagic-ci-cd/codemagic-docs#1614] [Announcements]
This thing is happening already for 6 months...
`
== Use Xcode 14.3 ==
> sudo xcode-select --switch /Applications/Xcode-14.3.app/Contents/Developer
Xcode 14.3
Build version 14E222b

== Use default version of CocoaPods ==
> gem list '^cocoapods$' --no-verbose
cocoapods (1.12.0)

== Use Java version 11 ==
> java -version
openjdk version "11.0.18" 2023-01-17 LTS
OpenJDK Runtime Environment Zulu11.62+17-CA (build 11.0.18+10-LTS)
OpenJDK 64-Bit Server VM Zulu11.62+17-CA (build 11.0.18+10-LTS, mixed mode)

== Use Node version latest ==
> n latest

Build failed :|
Unable to install Node version latest`

Build number: 645240f6bd4caabde325ef82

**Comment by icarusdust** [comment]
Hi @romanfurman6, it is about the node server as downloadable files are managed through Cloudflare, a bug could be causing this intermittent issue. I see that your subsequent builds succeeded.

**Comment by romanfurman6** [comment]
i see, so there is no way to fix this?

**Reply by icarusdust** [reply]
As it is something on the node side, we cannot do anything really but if the latest version is not required by your project, perhaps try the pre-installed version which is 19.7.0 by removing `node: latest` from codemagic.yaml and it will use it by default.

**Reply by romanfurman6** [reply]
Okay, I will try, thx 🙏
